### DISCUSSION TITLE
How to connect a VirtualBox client to LTSP

### DISCUSSION THREAD

**Discussion by rweilert** [ltsp/ltsp#616] [General]
[Just have to type everything from scratch only because I hadn't chosen a Category. Switching it deleted my whole text! I hate such bugs! Second go, I make it short:]

- There is an LTSP5 server with a lot of NUCs connected which PXE boot into it.
- There is one Win10 PC with a VirtualBox. The PC has a fixed IP within the LTSP range (.67.150).
- The VirtualBox is intended to boot from the server like one of the NUCs.

I've tried a lot of things including copying /boot/pxelinux.0 into $USER\.VirtualBox\TFTP\vmclient.pxe (when the vm is called "vmclient"), and it does load it but then fails loading the rest. I guess this is not what I want anyway, because it is intended to load everything from the server like any of the clients booting via PXE.

- Is it possible to do this anyway?
- If yes, which network type will I want, and how can I establish a standard PXE - DHCP - TFTP request?

Sorry if there are already answers to this here, I haven't found them. Just drop me a link to the discussion.

Thank you for any help!

Regards
Rolf

**Comment by alkisg** [comment]
LTSP5 isn't supported here, but from what I understood, you need to switch the VM networking from NAT to bridge mode.

**Reply by rweilert** [reply]
Yes, I guess the fact that it's an LTSP5 server is of no importance here. 

Now I tried bridged mode again, and now it runs. Thank you!

I just wonder how to tell VBox the size of the virtual monitor. Oh, I got it, I have to tweak it in Mate.

Regards
Rolf
